lambdas/CallbackLambdaFunction/index.js

```diff
@@ -48,13 +53,15 @@ function httpRequest(event, message, redirectCount) {
               reject(new Error('Too many redirects'));
             }
 
-            console.log(JSON.stringify({
-              msg: `Following HTTP redirect`,
-              location: res.headers.location,
-              count: redirectCount
-            }));
+            console.log(
+              JSON.stringify({
+                msg: `Following HTTP redirect`,
+                location: res.headers.location,
+                count: redirectCount,
+              }),
```

Comment:
Hmmph... trailing commas, you say.  Not my favorite setting, but a good linter makes everyone unhappy.